src/plugins/intel_gpu/src/kernel_selector/kernels/resample/resample_kernel_onnx.cpp

```diff
@@ -0,0 +1,171 @@
+// Copyright (C) 2018-2022 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "resample_kernel_onnx.h"
+#include <vector>
+#include <kernel_selector_utils.h>
+
+namespace kernel_selector {
+
+static constexpr size_t sub_group_size = 16;
+
+size_t ResampleKernelOnnx::GetOptimalBlockSize(const resample_params& params) const {
+    std::vector<size_t> block_width = { 16, 8, 4, 2, 1 };
+    for (auto& w : block_width)
+        if (params.outputs[0].X().v % w == 0)
+            return w;
```

Comment:
I would recommend using curly braces here as in the "GetOptimalDivisor" function.

Reply:
done